### PR TITLE
- Added geonetwork 3.2.2 and set it as the tip version, replacing 3.2.1

### PR DESCRIPTION
**Pull request by doublebyte1**
With this PR I add support to the latest release of GeoNetwork, [here](https://github.com/geonetwork/docker-geonetwork/commit/7ba5f533d9ee31cf1a22d4d6a1a84fbb74f86466).
This release fixes [this](https://github.com/geonetwork/docker-geonetwork/issues/4) issue.

### CODE REVIEW

File: library/geonetwork [hunk 1]
```diff
@@ -11,10 +11,10 @@ Tags: 3.0.5-postgres, 3.0-postgres
 GitCommit: e5cf5da76f557481c35b4a4d3e3cac77768a1302
 Directory: 3.0.5/postgres
 
-Tags: 3.2.1, 3.2, latest
-GitCommit: 507a1d8b02f15615566f7ac45662dbf3bc4d3649
-Directory: 3.2.1
+Tags: 3.2.2
```
Comment (yosifkit) [inline library/geonetwork]: Did you want to keep the `3.2` and `latest` tags here and the `3.2-postgres` and `postgres` tags below?

Comment (doublebyte1) [reply library/geonetwork]: @yosifkit Yes, I want to keep them: please check my latest [commit](https://github.com/docker-library/official-images/pull/3455/commits/e7bd2c24ba9c4ec659f7b8e4972845867f6569bd). Thanks for pointing that out! +1

**Comment by tianon** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/_bashbrew-list b/_bashbrew-list
index 8e62762..1280308 100644
--- a/_bashbrew-list
+++ b/_bashbrew-list
@@ -4,7 +4,7 @@ geonetwork:3.0.5
 geonetwork:3.0.5-postgres
 geonetwork:3.2
 geonetwork:3.2-postgres
-geonetwork:3.2.1
-geonetwork:3.2.1-postgres
+geonetwork:3.2.2
+geonetwork:3.2.2-postgres
 geonetwork:latest
 geonetwork:postgres
diff --git a/geonetwork_latest/Dockerfile b/geonetwork_latest/Dockerfile
index bbbc2d1..23c87b7 100644
--- a/geonetwork_latest/Dockerfile
+++ b/geonetwork_latest/Dockerfile
@@ -5,8 +5,8 @@ ENV DATA_DIR=$CATALINA_HOME/webapps/geonetwork/WEB-INF/data
 ENV JAVA_OPTS="-Djava.security.egd=file:/dev/./urandom -Djava.awt.headless=true -server -Xms512m -Xmx2024m -XX:NewSize=512m -XX:MaxNewSize=1024m -XX:PermSize=512m -XX:MaxPermSize=1024m -XX:+UseConcMarkSweepGC"
 
 #Environment variables
-ENV GN_VERSION 3.2.1
-ENV GN_DOWNLOAD_MD5 54140c3e4badc87651bd656f13f860f2
+ENV GN_VERSION 3.2.2
+ENV GN_DOWNLOAD_MD5 93b5279223503106ffeea3e9ca697805
 
 WORKDIR $CATALINA_HOME/webapps
 
diff --git a/geonetwork_postgres/Dockerfile b/geonetwork_postgres/Dockerfile
index e25c930..3c9458c 100644
--- a/geonetwork_postgres/Dockerfile
+++ b/geonetwork_postgres/Dockerfile
@@ -1,4 +1,4 @@
-FROM geonetwork:3.2.1
+FROM geonetwork:3.2.2
 
 RUN apt-get update && apt-get install -y postgresql-client && \
     rm -rf /var/lib/apt/lists/*
````

</details>

**Comment by tianon** [comment]
LGTM

Build test of #3455; e7bd2c24 (`geonetwork`):

```console

$ bashbrew build geonetwork:3.0.5
Using bashbrew/cache:9571bbd576e6c00ca50522c1dcac20b9147e148a11f28365d4da617ee6f83751 (geonetwork:3.0.5)
Tagging geonetwork:3.0.5
Tagging geonetwork:3.0

$ test/run.sh geonetwork:3.0.5
testing geonetwork:3.0.5
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.0.5-postgres
Using bashbrew/cache:5b064d77b268513459db148831af4f2826214c8d574dda1b61ca34c9a054132a (geonetwork:3.0.5-postgres)
Tagging geonetwork:3.0.5-postgres
Tagging geonetwork:3.0-postgres

$ test/run.sh geonetwork:3.0.5-postgres
testing geonetwork:3.0.5-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.2.2
Using bashbrew/cache:0cfaf4265ea636ca7eaa8434d63047ba39eb4357966061157d14951145d2566f (geonetwork:3.2.2)
Tagging geonetwork:3.2.2
Tagging geonetwork:3.2
Tagging geonetwork:latest

$ test/run.sh geonetwork:3.2.2
testing geonetwork:3.2.2
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build geonetwork:3.2.2-postgres
Using bashbrew/cache:b7ee87cefe88aad1162a6f3001d51106f798535afd040af4f65c244a6c11f2ce (geonetwork:3.2.2-postgres)
Tagging geonetwork:3.2.2-postgres
Tagging geonetwork:3.2-postgres
Tagging geonetwork:postgres

$ test/run.sh geonetwork:3.2.2-postgres
testing geonetwork:3.2.2-postgres
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**